tests/playbooks/tests_states.yml

```diff
@@ -47,3 +51,28 @@
             state: down
     - include_tasks: tasks/assert-device_absent.yml
     - include_tasks: tasks/assert-profile_absent.yml
+
+  pre_tasks:
+    - name: Save host state
+      tags:
+        - 'tests::tier1::cleanup'
+      import_tasks: "../save_state.yml"
+
+  post_tasks:
+    - name: Remove test profile
+      tags:
+        - 'tests::cleanup'
+        - 'tests::tier1::cleanup'
+        - 'tests::net::bridge::cleanup'
+      import_role:
+        name: linux-system-roles.network
+      vars:
+        network_connections:
+          - name: statebr
+            state: down
+            persistent_state: absent
```

Comment:
@i386x This fails, as the CI shows. You can not set anything on a nonexistent connection. (This is arguably a bug, because it means that the role is not properly idempotent, but for now the test must take this into account.)

Reply:
This is now tracked in #188